viz_scripts/scaffolding.py

```diff
@@ -317,23 +363,59 @@ def CO2_footprint_lb(df, distance, col):
     distance = distance in miles
     col = Replaced_mode or Mode_confirm
     """
+
+    conversion_lb_to_kilogram = 0.453592 # 1 lb = 0.453592 KG
+
     conditions_col = [(df[col+'_fuel'] =='gasoline'),
                        (df[col+'_fuel'] == 'diesel'),
                        (df[col+'_fuel'] == 'electric')]
-   
     gasoline_col = (df[distance]*df['ei_'+col]*0.000001)* df['CO2_'+col]
     diesel_col   = (df[distance]*df['ei_'+col]*0.000001)* df['CO2_'+col]
     electric_col = (((df[distance]*df['ei_'+col])+df['ei_trip_'+col])*0.001)*df['CO2_'+col]
 
     values_col = [gasoline_col,diesel_col,electric_col]
     df[col+'_lb_CO2'] = np.select(conditions_col, values_col)
+    df[col+'_kg_CO2'] = df[col+'_lb_CO2'] * conversion_lb_to_kilogram
```

Comment:
nit: I understand why you are doing this, but there is a mismatch between the name `footprint_lb` and what is computes. I would suggest renaming when revisiting the entire computation.

Reply:
Renamed this to `footprint_default`. Considering we'll have two different mapping - one from `auxiliary_files/energy_intensity.csv` and other from the `config` file where `label_options` is available.
Renamed the one calculating with dynamic config with suffix - `footprint_dynamic`.
Both `footprint_default` and `footprint_dynamic` will have `kg/km` and `lb/mile` units.

Reply:
I don't understand the math here 

I assume that `lb_CO2` is $\frac{lb}{mile}$ per your previous comment.
your conversion gets it to $\frac{kg}{mile}$
where is the conversion to $\frac{kg}{km}$?

Reply:
Correction to my previous comment. 
Deduction of the units from above calculation:
Considering example for `electric_col`
```
electric_col = (((df[distance]*df['ei_'+col])+df['ei_trip_'+col])*0.001)*df['CO2_'+col]
```
df[distance] - miles
col = Mode_confirm/ Replaced_mode
df[ei_Mode_confirm] - kWH/PMT [kWH/ Passenger-mile]

df[ei_trip_Mode_confirm] - kWH/Trip

df[CO2_Mode_confirm] - lbs/MWH

This forms the following:
( mile * kWH/ Passenger-mile + kWH/Trip ) * 0.001 * lbs/MWH
= ( kWH/ Passenger + kWH/Trip ) * lbs/kWH
= kWH/ Trip * lbs/kWH
= lbs / Trip

Referring to this definition of PMT from [National Household Travel Survey](https://nhts.ornl.gov/2009/pub/usersguideglossary.pdf), PMT = Person Miles of Travel.
PMT: The number of miles traveled by each person on a trip.

The final unit of lbs_CO2 is lbs/ Trip. 
Therefore, the value of lb_CO2 computation is independent of distance unit i.e. either `miles` or `km`.